ctapipe_io_lst/tests/test_multifile.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from pathlib import Path
+import os
+
+test_data = Path(os.getenv('LSTCHAIN_TEST_DATA', 'test_data')).absolute()
+test_r0_dir = test_data / 'real/R0/20200218'
+
+
+def test_multifile():
+    from ctapipe_io_lst.multifiles import MultiFiles
+
+    paths = test_r0_dir.glob('LST-1.*.Run02005.0000_first50.fits.fz')
+
+    with MultiFiles(paths) as multi_files:
+
+        assert len(multi_files) == 200
+        assert multi_files.num_inputs() == 4
+
+        event_count = 0
+        for event in multi_files:
+            event_count += 1
+            assert event.event_id == event_count
+
+        assert event_count == 200
+
+
+def test_rewind():
+    from ctapipe_io_lst.multifiles import MultiFiles
+
+    paths = test_r0_dir.glob('LST-1.*.Run02005.0000_first50.fits.fz')
+
+    with MultiFiles(paths) as multi_files:
+
+        for expected_event_id in range(1, 10):
+            event = next(multi_files)
+            assert event.event_id == expected_event_id
```

Comment:
this line in principle could be skipped as well, since it is not testing the rewinding, but rather normal reading of the file (and exactly the same test is done in the function above)

Reply:
It checks that after "rewinding" the normal iteration works as expected. This is clearly not redundant, this is is exactly what failed before I fixed "rewind"

Reply:
please check my comment again, I've explicitly meant this line 36 and not the whole function. You read first 10 events checking that the numbers increase, then rewind and then read them again checking again the numbers. The second check (in line 42) is perfectly valid, but if assert of line 36 fails it would also fail in line 22, hence I think you could just read the 10 events (without checking) then rewind and read them again (this time with checking). But as I wrote above this is just a redundant line, so if you want to keep it, it is fine with me